nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -936,9 +936,10 @@ In addition to numerous new and upgraded packages, this release has the followin
   package.
 
 - The option
-  [services.networking.networkmanager.enableFccUnlock](#opt-networking.networkmanager.enableFccUnlock)
-  was added to support FCC unlock procedures. Since release 1.18.4, the ModemManager
-  daemon no longer automatically performs the FCC unlock procedure by default. See
+  [services.networking.networkmanager.enableBundledFccUnlockScripts](#opt-networking.networkmanager.enableBundledFccUnlockScripts)
+  was added to support FCC unlock procedures.
+  Since release 1.18.4, the ModemManager daemon no longer automatically performs
+  the FCC unlock procedure by default. See
```

Comment:
We shouldn't modify old release notes.

Reply:
This is required in order not to break link checker.

Reply:
I removed the hyperlink, put it in backticks, and squashed the two lines together.